ide/static/js/content.js

```diff
@@ -411,6 +414,27 @@ class Content extends React.Component {
             downloadAnchor.download = response.name;
             downloadAnchor.href = response.url;
             downloadAnchor.click();
+            if ('customLayers' in response && response.customLayers.length !== 0) {
+              this.addInfo(
+                <span>
```

Comment:
Can we handle this formatting in the `info` component instead of putting it in `content.js`.